modin/pandas/general.py

```diff
@@ -454,6 +462,18 @@ def concat(
         ).index
     else:
         new_idx = None
+
+    if len(list_of_objs) == 0:
+        idx = input_list_of_objs[0].index.append(
+            [f.index for f in input_list_of_objs[1:]]
+        )
+        if axis == 1 or any(
+            isinstance(obj, (DataFrame, pandas.DataFrame)) for obj in input_list_of_objs
+        ):
+            return DataFrame(index=idx)
+        else:
+            return Series(index=idx)
```

Comment:
This case should not be here. Do you have a specific example when we need to explicitly create a Series here?

Reply:
As I mentioned here https://github.com/modin-project/modin/pull/5673#discussion_r1115074467, this case is described in the documentation.

Reply:
in the only case when we need to return a series, we have a separate branch with return statement. See https://github.com/modin-project/modin/pull/5673#discussion_r1115126900 and https://github.com/modin-project/modin/pull/5673#discussion_r1116179037

Reply:
You are right. This branch is dead. I've removed it. Thanks.